app/src/main/java/com/guerinet/mymartlet/util/dbflow/databases/CoursesDB.java

```diff
@@ -46,15 +47,12 @@
      * @param callback Optional callback called when the transaction is finished
      */
     public static void setCourses(Term term, List<Course> courses,
-            @Nullable DBUtils.Callback callback) {
-        if (courses == null) {
-            courses = new ArrayList<>();
-        }
+                                  @Nullable DBUtils.Callback callback) {
+
+        if (courses == null) courses = new ArrayList<>();
 
         // Set the term on the passed list of courses
-        for (Course course : courses) {
-            course.setTerm(term);
-        }
+        courses.forEach(course -> course.setTerm(term));
```

Comment:
From what I understand, your purpose of using [retrolambda](https://github.com/jguerinet/MyMartlet/blob/master/app/build.gradle#L35) is so that you compile to previous Java versions.

You can specify the output bytecode version like [so](https://github.com/ctf/CTF-Android/blob/master/CTF/build.gradle#L106)

I don't think this will be related to Android versions